<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterSeller</class>
 <widget class="QWidget" name="RegisterSeller">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="MarketName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>-50</y>
     <width>401</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Showcard Gothic</family>
     <pointsize>36</pointsize>
     <bold>false</bold>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="text">
    <string>Registration</string>
   </property>
  </widget>
  <widget class="QLabel" name="MarketName_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>0</y>
     <width>401</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Showcard Gothic</family>
     <pointsize>36</pointsize>
     <bold>false</bold>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="text">
    <string>as</string>
   </property>
  </widget>
  <widget class="QLabel" name="MarketName_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>0</y>
     <width>401</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Showcard Gothic</family>
     <pointsize>36</pointsize>
     <bold>false</bold>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="text">
    <string>Seller</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditEmail">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
                        {
                        border-width: 2px;
                        border-radius: 5px;
                        }
                        QLineEdit:Focus {
                        border:1px solid #6443DA
                        }
                    </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>16</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::EchoMode::Normal</enum>
   </property>
   <property name="placeholderText">
    <string>   Почта</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditNum">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>230</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
                        {
                        border-width: 2px;
                        border-radius: 5px;
                        }
                        QLineEdit:Focus {
                        border:1px solid #6443DA
                        }
                    </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>16</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::EchoMode::Normal</enum>
   </property>
   <property name="placeholderText">
    <string>   Номер телефона</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditLogin">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
                        {
                        border-width: 2px;
                        border-radius: 5px;
                        }
                        QLineEdit:Focus {
                        border:1px solid #6443DA
                        }
                    </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>   Логин</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPassword_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>290</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
                        {
                        border-width: 2px;
                        border-radius: 5px;
                        }
                        QLineEdit:Focus {
                        border:1px solid #6443DA
                        }
                    </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>16</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::EchoMode::Password</enum>
   </property>
   <property name="placeholderText">
    <string>   Подтвердите пароль</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPassword_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>350</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
                        {
                        border-width: 2px;
                        border-radius: 5px;
                        }
                        QLineEdit:Focus {
                        border:1px solid #6443DA
                        }
                    </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>16</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::EchoMode::Password</enum>
   </property>
   <property name="placeholderText">
    <string>   Пароль</string>
   </property>
  </widget>
  <widget class="QPushButton" name="registerButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>470</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>UpArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">    QPushButton {
        background-color: #4BB34B;
        color: white;
        border-radius: 8px;
        padding: 10px 20px;
    }
    QPushButton:hover {
        background-color: #5fd35f;
    }
    QPushButton:pressed {
        background-color: #3fa53f;
    }</string>
   </property>
   <property name="text">
    <string>Создать аккаунт</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>420</y>
     <width>311</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Я НЕ БУДУ ПРОДАВАТЬ ЗАПРЕЩЕННЫЕ ТОВАРЫ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>530</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">    QPushButton {
        background-color: #3b5249;
        color: white;
        border-radius: 8px;
        padding: 10px 20px;
    }
    QPushButton:hover {
        background-color: #5fd35f;
    }
    QPushButton:pressed {
        background-color: #3fa53f;
    }</string>
   </property>
   <property name="text">
    <string>Я больше не хочу быть продавцом</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
